front/components/spaces/ContentActions.tsx

```diff
@@ -42,6 +42,11 @@ export type ContentAction = {
   contentNode?: DataSourceViewContentNode;
 };
 
+const isUploadOrEditAction = (
+  action: ContentActionKey | undefined
+): action is "DocumentUploadOrEdit" | "TableUploadOrEdit" =>
```

Comment:
`nit` can we create a type to avoid those magic values.

Reply:
done